### ISSUE TITLE
Which Settings recommended for proxies?

### ISSUE THREAD

**Issue by Marow** [hartleybrody/public-amazon-crawler#12]
Hi, i tried some proxies cause that much requests costs very much with the time.

So i tried like that site http://pubproxy.com/#premium but which settings are needed for this crawler?
I have proxybonanze already, working fine with  SOCKS5, so i tried so much on pubproxy but its not working...

**Comment by rochos-foniem** [comment]
I tried pubproxy but their proxies are blocked and their system is not compatible with the script, then I tried proxycrawl and it worked perfectly with the crawler, I used their API so you just have to replace amazon urls with the api url and you can continue using the script to crawl amazon

**Comment by rochenka** [comment]
yes I got similar issues and solved with this service https://proxycrawl.com/scraping-api-avoid-captchas-blocks I also used the proxycrawl python package.
